sdk/storage/azure-storage-blob/README.md

```diff
@@ -312,6 +313,24 @@ for (BlobItem blobItem : blobContainerClient.listBlobs()) {
 }
 ```
 
+### Copy a blob
+
+Copying a blob. Please refer to the javadocs on each of these methods for more information around requirements on the 
+copy source and its authentication.
+
+<!-- embedme ./src/samples/java/com/azure/storage/blob/ReadmeSamples.java#L131-L132 -->
+```java
+SyncPoller<BlobCopyInfo, Void> poller = blobClient.beginCopy("<url-to-blob>", Duration.ofSeconds(1));
```

Comment:
Should we maybe add like if you get a CannotVerifyCopySource, make sure authentication is correct?

Reply:
Spoke offline. It's atypical for us to call out specific error codes in our docs at the moment, so I'll leave this out for now. If these updates still are insufficient to help customers solve their own issues, we can add in call-outs to that error code later.